set default graphics mode to text

Copyright (C) 2014 Curt Brune <curt@cumulusnetworks.com>

SPDX-License-Identifier:     GPL-2.0

For EFI set the default graphics mode to text.

This supresses an ugly, but harmless message about no video mode
found.

diff --git a/grub-core/loader/i386/linux.c b/grub-core/loader/i386/linux.c
index 31fb91e..b1f34f8 100644
--- a/grub-core/loader/i386/linux.c
+++ b/grub-core/loader/i386/linux.c
@@ -45,8 +45,8 @@ GRUB_MOD_LICENSE ("GPLv3+");
 #ifdef GRUB_MACHINE_EFI
 #include <grub/efi/efi.h>
 #define HAS_VGA_TEXT 0
-#define DEFAULT_VIDEO_MODE "auto"
-#define ACCEPTS_PURE_TEXT 0
+#define DEFAULT_VIDEO_MODE "text"
+#define ACCEPTS_PURE_TEXT 1
 #elif defined (GRUB_MACHINE_IEEE1275)
 #include <grub/ieee1275/ieee1275.h>
 #define HAS_VGA_TEXT 0
